src/Controls/tests/UITests/Tests/Issues/Issue3525.cs

```diff
@@ -18,9 +18,7 @@ public Issue3525(TestDevice device)
 	[Test]
 	public void SpanRegionClicking()
 	{
-		if (Device == TestDevice.Mac ||
-			Device == TestDevice.iOS ||
-			Device == TestDevice.Windows)
+		if (Device == TestDevice.Windows)
 		{
 			Assert.Ignore("This test is failing on iOS/Mac Catalyst/Windows because the feature is not yet implemented: https://github.com/dotnet/maui/issues/4734");
```

Comment:
```suggestion
			Assert.Ignore("This test is failing on Windows because the feature is not yet implemented: https://github.com/dotnet/maui/pull/17731");
```